ci.js

```diff
@@ -120,6 +120,17 @@ async function getResults(driver) {
   return {specResults: flatten(specResults), failedSuiteResults: flatten(failedSuiteResults)};
 }
 
+function cleanup() {
+  return Promise.resolve().then(() => {
```

Comment:
What's the reason for this wrapping `Promise.resolve().then`? I would expect just `Promise.all().catch` to work correctly here.

Reply:
We could probably scrap that, actually...

Originally I couldn't tell what was going wrong, because the error message was getting lost in the ether, so I wrapped the whole thing.  Then afterwards I realized the issue was driver wasn't defined yet, and added the inline checks.

TLDR: both checks are too much and could be simplified now.

Reply:
Sweet. Please update to include the simplified checks. Thanks!